fabric-climbable-api-v1/src/main/resources/fabric.mod.json

```diff
@@ -0,0 +1,27 @@
+{
+  "schemaVersion": 1,
+  "id": "fabric-climbable-api-v1",
```

Comment:
Should this still be considered an api? This no longer exposes any API, but rather does some implementation tweaks like the crash reports or indigo.

Reply:
IMO a json API is still an API. As modmuss pointed out on discord, those modules aren't things mods interface with, while a tag is.